pkg/fleet-manager/application/rollout_helper_test.go

```diff
@@ -0,0 +1,653 @@
+/*
+Copyright Kurator Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package application
+
+import (
+	"context"
+	"fmt"
+	"reflect"
+	"testing"
+
+	flaggerv1b1 "github.com/fluxcd/flagger/pkg/apis/flagger/v1beta1"
+	"github.com/fluxcd/flagger/pkg/apis/istio/common/v1alpha1"
+	istiov1alpha3 "github.com/fluxcd/flagger/pkg/apis/istio/v1alpha3"
+	appsv1 "k8s.io/api/apps/v1"
+	autoscalingv1 "k8s.io/api/autoscaling/v1"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/intstr"
+	applicationapi "kurator.dev/kurator/pkg/apis/apps/v1alpha1"
+)
+
+var sign = true
+var timeout = 50
+var int32Time = int32(50)
+var min = 99.0
+var max = 500.0
+var rolloutPolicy = applicationapi.RolloutConfig{
```

Comment:
reduce pkg level vars as much as possible

Reply:
Now I use a function to create a rollout policy, reduce pkg level vars.